CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## v0.19.1 (2018-3-13)
+
+### Backward compatibility with the legacy test syntax has been restored ([#2210](https://github.com/DevExpress/testcafe/issues/2210))
+
+### Bug Fixes
+
+* The `document.all` property is now overridden ([testcafe-hammerhead/#1046](https://github.com/DevExpress/testcafe-hammerhead/issues/1046))
+* Proxying properties in `async` class methods is now supported ([testcafe-hammerhead/#1510](https://github.com/DevExpress/testcafe-hammerhead/issues/1510))
+* Fixed wrong proxying of a `localStorage` check in WebWorkers ([testcafe-hammerhead/#1510](https://github.com/DevExpress/testcafe-hammerhead/issues/1510))
+* Function wrappers no longer break *is-defined* checks ([testcafe-hammerhead/#1496](https://github.com/DevExpress/testcafe-hammerhead/issues/1496))
+
```

Comment:
Dublicate for testcafe-hammerhead/#1510.

The final changelog:
* The `document.all` property is now overridden ([testcafe-hammerhead/#1046](https://github.com/DevExpress/testcafe-hammerhead/issues/1046))
* Proxying properties in `async` class methods is now supported ([testcafe-hammerhead/#1510](https://github.com/DevExpress/testcafe-hammerhead/issues/1510))
* Fixed wrong proxying of a `localStorage` check in WebWorkers ([testcafe-hammerhead/#1496](https://github.com/DevExpress/testcafe-hammerhead/issues/1496))

Reply:
Oh, I thought you duplicated them deliberately https://github.com/DevExpress/testcafe/pull/2211#discussion_r174173455

Reply:
Yes, it's my mistake.